src/main/kotlin/com/neva/gradle/fork/config/Config.kt

```diff
@@ -14,13 +17,16 @@ import java.io.FileInputStream
 import java.io.FileOutputStream
 import java.util.*
 
-abstract class Config(val project: Project, val name: String) {
+abstract class Config(val project: Project, private val propertyDefinitions: PropertyDefinitions, val name: String) {
 
-  val prompts = mutableMapOf<String, PropertyPrompt>()
+  private val prompts = mutableMapOf<String, PropertyPrompt>()
 
-  val props by lazy { promptFill() }
+  private val props by lazy { promptFill() }
 
-  val rules = mutableListOf<Rule>()
+  private val rules = mutableListOf<Rule>()
+
+  val properties: List<Property>
+    get() = this.prompts.values.map { prompt -> propertyDefinitions.getProperty(prompt) }
```

Comment:
I dislike getter looking methods which are not getters actually. if getter looking method has a param, then I am changing it to less confusing/more precise `find` `determine` methods etc